### DISCUSSION TITLE
Connect Route53 nameservers to vercel nameservers

### DISCUSSION THREAD

**Discussion by bstillwagon-mylo** [vercel/community#3185] [Help] [answered]
### Summary

I'm trying to use the Preview Deployment Suffix feature to change the suffix to one of our already defined nameservers in Route53 (route53 is used to manage all our DNS records).  When I try to update the suffix, I get the error saying it must match a vercel nameserver.

### Steps to Reproduce

<img width="949" alt="Screenshot 2023-07-10 at 4 06 31 PM" src="https://github.com/vercel/community/assets/19784499/dcf4b94b-1b29-4b73-bd9d-99ad56f41608">

<img width="939" alt="Screenshot 2023-07-10 at 4 35 27 PM" src="https://github.com/vercel/community/assets/19784499/17b6079e-f4a8-4802-b3c3-783713c6c261">

I have turned on the Vercel DNS as well trying to get this to work, but once turned on, I'm still not able to get the options in the below: Nameservers section to be enabled.  I want to keep my records managed in Route53, but is there any way to make this work that way?

<img width="1262" alt="Screenshot 2023-07-10 at 4 11 07 PM" src="https://github.com/vercel/community/assets/19784499/d54f382e-3d82-4054-88cc-43329f985786">

**Comment by smaeda-ks** [comment, marked as answer]
@bstillwagon-mylo try this route: https://vercel.com/guides/preview-deployment-suffix-without-vercel-nameservers
